product_configurator_purchase/views/purchase_views.xml

```diff
@@ -21,9 +21,10 @@
                         states="draft,sent"
                         class="oe_highlight"
                         type="action"
-                        style="margin-top: 15px;margin-bottom: 10px;"
+                        style="margin-top: 15px;margin-bottom: 10px;margin-left: 20px;"
```

Comment:
I made changes related to allignment of buttons in purchase_note. This change is to create space between buttons 
![buttons](https://user-images.githubusercontent.com/28155989/28215565-7ffc0f06-687c-11e7-8d5d-75dc0dd88195.png)